workflow/envs/canu.yaml

```diff
@@ -2,5 +2,6 @@ channels:
   - bioconda
   - conda-forge
 dependencies:
-  - canu =2.2
-  - minimap2 =2.22
+  # do not use canu 2.2 !
```

Comment:
what's the problem with the newer version?

Reply:
It is not doing the job. It will just not correct the reads. Change of version completely broke the de-novo assembly part of the rule (which in my opininon we should drop anyway, but thats another discussion). But what's worse: It doesn`t give an error, but just outputs uncorrected reads as corrected. The only hint is one half-sentence, somewhere hidden in the ten-gigatons of report-output it craps out (that was also the reason why I intentionally did not refer it to the log, because I wanted to keep an eye on it during developmnet, in case version was changed again). It was quite a nightmare to find out what is going on there. Why it is happening I don´t know exactly, but I suspect that between versions a read-length or overlap threshold was hardcoded internally, which cannot be overwritten by the command-line arguments anymore (where I did specify suitable overlap and read length parameters).